src/utils/systemJsRendering.ts

```diff
@@ -16,7 +16,8 @@ export function getSystemExportStatement(
 			exportNamesByVariable.get(variable)![0]
 		)},${_}${variable.getName(getPropertyAccess)}${modifier})`;
 	} else {
-		const fields: [key: string, value: string][] = [];
+		const fields: [key: string | null, value: string][] = [];
+		fields.push([null, `__proto__:${_}null`]);
```

Comment:
I do not unterstand why we need this. This generates a lot of noise and if we set a potential __proto__ variable via `["__proto__"]`, this should not have any effect.